punchcut/src/sink.rs

```diff
@@ -0,0 +1,74 @@
+use peniko::kurbo::{BezPath, PathEl};
```

Comment:
We are going to remove the peniko and kurbo deps iiuc, so presumably this needs to significantly change?

Reply:
I'll just drop the trait impls from that module. We can potentially add them back behind a feature gate in the future, but it's not strictly important.